misc-utils/cal.c

```diff
@@ -552,6 +554,7 @@ int main(int argc, char **argv)
 			ctl.req.month = MONTHS_IN_YEAR < m ? 1 : m;
 	}
 
+	init_weekdays(&ctl);
 	headers_init(&ctl);
 
```

Comment:
Cannot we use the same concept for all the function names? :-)    
`init_weekdays() init_headers()` or `weekdays_init() headers_init()`